src/tigerbeetle/main.zig

```diff
@@ -487,7 +479,6 @@ fn command_start(
 
     while (true) {
         replica.tick();
-        if (multiversion != null) multiversion.?.tick();
```

Comment:
Removing this one line is the entire backstory behind my recent multiversion yak shave! 🤣